src/ui/tools/Tags.js

```diff
@@ -56,6 +56,10 @@ ui.defaultTags = {
 	input: {
 		input: "input",
 	},
+	mapbox: {
+		container: "div",
```

Comment:
Using just a **div** tag name isn't allowed in our UI design pattern, it must be **m-mapbox-wrapper** for this instance example. We don't use regular **div** tags for semantic purposes.